misc/matomo.md

```diff
@@ -10,35 +10,21 @@ the dockerfile and config is located on [GitHub](https://github.com/MITLibraries
 
 This guide will walk through the basic steps to add a website to Matomo. 
 For more in-depth information on using and developing Matomo, see the 
-[offical Matomo docs](https://matomo.org/help/).
+[offical Matomo docs](https://matomo.org/help/). For information specific
+to our implementation, such as how to access staging and prod, visit
+the [Matomo analytics](https://wikis.mit.edu/confluence/display/UXWS/Matomo+analytics) page in the UXWS Confluence.
 
-## Universal settings
+## Privacy settings
 
 All websites tracked by Matomo anonymize visitor IPs to two bytes (e.g., 
-192.168.x.x).
+192.168.x.x). This is a universal setting that is applied to all new
+properties, and it ensures that we do not store personally identifiable
+information (PII) in Matomo.
 
 ## Setting up a new website in Matomo
 
-Each app is tracked as a website in Matomo. Please create a different 
-website for each environment of your application (e.g., staging and prod 
-will be separate websites in Matomo).
-
-_Note: Accounts with superuser permissions add new websites to Matomo. 
-If you don’t have superuser access, ask EngX._
-
-1. Log into Matomo at https://analytics-prod.mitlib.net. (There is also 
-a staging instance behind the Cisco VPN at https://analytics-stage.mitlib.net.) 
-If you don't have an account, EngX can create one for you.
-2. Click the Settings link (gearbox in navbar) > Websites > Manage > 
-Add a new measurable,
-3. Give the measurable a useful name. We generally use the name the app is 
-known by internally, plus a note if it’s the staging app. E.g.: “Bento” 
-or “Bento (staging)”.
-4. Add the root URL for the application, as well as any aliases. It is 
-not necessary to add specific routes in the application.
-5. Leave the remaining settings as their defaults, then click ‘Save’.
-
-The website will now be listed under Websites > Manage.
+See [the official Matomo documentation](https://matomo.org/faq/how-to/create-and-manage-websites/). Note that only the
```

Comment:
Can you clarify this means what it says? Other accounts can access matomo but I suspect this is trying to convey that only superuser can do a specific thing?